printer_mods/Mckurt/Switchwire Y Belt Tensioner/Readme.md

```diff
@@ -0,0 +1,24 @@
+| Creator | Config title | Description | V0 | V1.8 | V2.4 | VSW |
+| MCKURT11 | Switchwire Y Belt Tensioner | :x: | :x: | :x: | :x: | :heavy_check_mark: |
+
+![Screenshot_41](https://user-images.githubusercontent.com/65568463/132855469-595db6d7-ff85-4a1b-8bbc-395d22cf8505.png)
+![Screenshot_43](https://user-images.githubusercontent.com/65568463/132855517-7a9b16e8-dce0-4678-a239-b435fcceac93.png)
+
+This mod for Voron Switchwires Y axis Belt tension, you dont need cut panels
+
+Fasteners
+
+2x M3x8mm SHCS
+2x M5x10mm SHCS
+1x M5x10mm BHCS
+1x M5x16 up to 20mm for tensioner
+2x M5 T-Nut (For 3030)
+1x M5 T-Nut (For 2020)
+2x M3 Threaded Insert
+
+Use Allen key for tension your belt,
+You can use brim while printing "Body" part
+
+Have a good prints (:
+
+Photos might be differences then STLs, because of upgrades at latest and tested parts are Ready for your download
```

Comment:
This sentence is better written as

    Photos might be different from STLs because of updates.  Latest and tested parts are ready for your download.